.github/scripts/promote_branch.sh

```diff
@@ -132,7 +132,6 @@ fi
 # A change cannot go into staging if staging and production differ
 if [[ "${TARGET_BRANCH}" == "staging" && "${FORCE_TO_STAGING}" != "true" ]] ; then
     git checkout origin/staging
-    update_taskGitRevision_in_pipelines production
```

Comment:
This was just a leftover from before we made changes that allows us to completely avoid any changes, right? Just checking. It seems there is no such function anymore. It was a function, right?

Reply:
Yep correct on both fronts